src/components/card/card.spec.tsx

```diff
@@ -40,15 +52,37 @@ describe("Card", () => {
     "large",
   ])(
     "when spacing prop is set to %s, CardRow and CardFooter children should have a spacing prop injected with same value",
+    (spacing) => {
```

Comment:
**nitpick**: would it be worth modifying the test description slightly here, since `spacing` is no longer injected?

Reply:
Agree this can be tidied up slightly 👍